### ISSUE TITLE
Modified model

### ISSUE THREAD

**Issue by agnieszkajust** [csbhr/CDVD-TSP#20]
How should I change class Decoder if I want to learn Network with additional:
torch.nn.Conv2d(in_channels=16, out_channels=16, kernel_size=3, stride=1, padding=1),
torch.nn.LeakyReLU(inplace=False, negative_slope=0.1) 
in each module in Extractor.

**Comment by Etienne66** [comment]
This is part of https://github.com/sniklaus/pytorch-pwc and you would need to not load the pre-trained model `network-default.pytorch` which it loads into the `Flow_PWC` model. Also the `in_channels` would be `intCurrent + 128 + 128 + 96 + 64 + 32` with `out_channels=16`. There are several other places you would need to adjust the `in_channels`

It is still trainable without the pre-trained model for the flow.
